Resources/public/js/views/tabs/ez-locationviewviewdetailstabview.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-locationviewviewdetailstabview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Location View View Tab view class.
+     *
+     * @module ez-locationviewviewdetailstabview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Location View View Details tab class.
+     *
+     * @namespace eZ
+     * @class LocationViewViewDetailsTabView
+     * @constructor
+     * @extends eZ.LocationViewTabView
+     */
+    Y.eZ.LocationViewViewDetailsTabView = Y.Base.create('locationViewViewDetailsTabView', Y.eZ.LocationViewTabView, [], {
+        render: function () {
+            var container = this.get('container');
+
+            container.setHTML(this.template({
+                "currentVersion": this.get('content').get('currentVersion').get('versionNo'),
+                "locationId": this.get('location').get('locationId'),
```

Comment:
to make this simpler, I would pass `this.get('content').toJSON()`, `this.get('location').toJSON()` and `this.get('content').get('currentVersion').toJSON()` to the template.